{"name": "test_authentication_error_anthropic", "status": "passed", "description": "Test Anthropic authentication error produces standard format.", "attachments": [{"name": "log", "source": "3149bc15-0630-48a4-acfd-fa4073259494-attachment.txt", "type": "text/plain"}], "start": 1757668162417, "stop": 1757668162418, "uuid": "b93f3703-781a-4ed5-97e2-4c45d374a703", "historyId": "4cf8759afeb15f6d713e8628096554ed", "testCaseId": "4cf8759afeb15f6d713e8628096554ed", "fullName": "tests.unit.test_standardized_llm_errors.TestStandardizedErrors#test_authentication_error_anthropic", "labels": [{"name": "tag", "value": "asyncio"}, {"name": "parentSuite", "value": "tests.unit"}, {"name": "suite", "value": "test_standardized_llm_errors"}, {"name": "subSuite", "value": "TestStandardizedErrors"}, {"name": "host", "value": "Joshuas-MacBook-Pro-2.local"}, {"name": "thread", "value": "34209-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.unit.test_standardized_llm_errors"}], "titlePath": ["tests", "unit", "test_standardized_llm_errors.py", "TestStandardizedErrors"]}